src/Orleans.Core/Serialization/IExternalSerializer.cs

```diff
@@ -13,12 +12,6 @@ namespace Orleans.Serialization
     public interface IExternalSerializer
```

Comment:
This is a breaking change, we need to make sure that it's called out in the release notes.

Reply:
@jdom to notify